src/commands/environment/environment-builder.ts

```diff
@@ -8,11 +10,15 @@ export default function environmentCommandBuilder() {
         .description('Get current environment')
         .asyncAction(Env.getEnv);
 
+    // todo: remove in 4.0.6
     env.command('ls')
         .description('List supported environments')
-        .asyncAction(Env.listEnvs);
+        .action(()=>{
+            console.warn(chalk.yellow(`Warning: The "env ls" command is deprecated. Ref: ${getPlatformUrls().partners}/help/docs/partners/themes/vuejs/command-reference#environment-commands-1`));
+        });
 
     env.command('set')
+        // todo: remove -n option in 4.0.6
         .option('-n, --name [env-name]', 'Environment name')
```

Comment:
@manishprajapati52 @jinalvirani aren't we removing env set option here ?

Reply:
To show deprecation warning we have kept the option as it is. If you check the logic, it has condition to show env set -n command is deprecated.